admin_guide/topics/proc_etcd-quorum-single-node-v2-v3.adoc

```diff
@@ -15,10 +15,13 @@ remaining etcd node a standalone etcd cluster.
 [discrete]
 == Procedure
 
-. On the etcd node that you did not remove from the cluster, stop all etcd services:
+. On the etcd node that you did not remove from the cluster, stop all etcd services by 
+removing the etcd pod definition and rebooting the host:
 +
 ----
-# systemctl stop etcd.service
+# mkdir -p /etc/origin/node/pods-stopped
+# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
+# reboot
```

Comment:
We need to restore files before reboot and to restore files we need to stop atomic-openshift-node.service (otherwise it starts again).

```
# mkdir -p /etc/origin/node/pods-stopped
# mv /etc/origin/node/pods/* /etc/origin/node/pods-stopped/
# systemctl stop atomic-openshift-node
# mv /etc/origin/node/pods-stopped/* /etc/origin/node/pods/
# reboot
```

We probably need more steps to see if etcd is properly stopped. I'll send a query to openshift-sme.

Reply:
Seems we just need stop etcd service here, so I think should move etcd.yaml instead of all files in `/etc/origin/node/pods/` folder unless you want stop master-api and controllers too. 
so it might be:
`mv /etc/origin/node/pods/etcd.yaml /etc/origin/node/pods-stopped/`
@xingxingxia @geliu2016 could you help confirm?

Reply:
@lihongan yes, AGREE.